examples/voice_receive/go.mod

```diff
@@ -1,7 +1,60 @@
 module github.com/bwmarrin/discordgo/examples/voice_receive
 
+go 1.22.4
+
 require (
-	github.com/bwmarrin/discordgo v0.21.1
+	github.com/bwmarrin/discordgo v0.28.1
 	github.com/pion/rtp v1.6.0
 	github.com/pion/webrtc/v3 v3.0.0-20200721060053-ca3cc9d940bc
 )
+
+require (
+	github.com/cheekybits/genny v1.0.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
+	github.com/golang/mock v1.2.0 // indirect
+	github.com/golang/protobuf v1.2.0 // indirect
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/hpcloud/tail v1.0.0 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/kr/pty v1.1.1 // indirect
+	github.com/kr/text v0.1.0 // indirect
+	github.com/lucas-clemente/quic-go v0.7.1-0.20190401152353-907071221cf9 // indirect
+	github.com/marten-seemann/qtls v0.2.3 // indirect
+	github.com/onsi/ginkgo v1.7.0 // indirect
+	github.com/onsi/gomega v1.4.3 // indirect
+	github.com/pion/datachannel v1.4.19 // indirect
+	github.com/pion/dtls/v2 v2.0.2 // indirect
+	github.com/pion/ice/v2 v2.0.0-rc.8 // indirect
+	github.com/pion/logging v0.2.2 // indirect
+	github.com/pion/mdns v0.0.4 // indirect
+	github.com/pion/quic v0.1.1 // indirect
+	github.com/pion/randutil v0.1.0 // indirect
+	github.com/pion/rtcp v1.2.3 // indirect
+	github.com/pion/sctp v1.7.8 // indirect
+	github.com/pion/sdp/v2 v2.4.0 // indirect
+	github.com/pion/srtp v1.5.0 // indirect
+	github.com/pion/stun v0.3.5 // indirect
+	github.com/pion/transport v0.10.1 // indirect
+	github.com/pion/turn/v2 v2.0.4 // indirect
+	github.com/pion/udp v0.1.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/sclevine/agouti v3.0.0+incompatible // indirect
+	github.com/stretchr/objx v0.1.0 // indirect
+	github.com/stretchr/testify v1.6.1 // indirect
+	golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b // indirect
+	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 // indirect
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
+	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 // indirect
+	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1 // indirect
+	golang.org/x/text v0.3.3 // indirect
+	golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e // indirect
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/fsnotify.v1 v1.4.7 // indirect
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+)
```

Comment:
I'm not exactly sure where all these extra dependencies came from.
After running `go mod tidy` on `go.mod` with updated versions, it results only in:
```
module github.com/bwmarrin/discordgo/examples/voice_receive

go 1.22.4

require (
	github.com/bwmarrin/discordgo v0.28.1
	github.com/pion/rtp v1.6.0
	github.com/pion/webrtc/v3 v3.0.0-20200721060053-ca3cc9d940bc
)

require (
	github.com/gorilla/websocket v1.4.2 // indirect
	github.com/pion/randutil v0.1.0 // indirect
	golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b // indirect
	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 // indirect
)
```